### PR TITLE
Metrics interval fixes

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

**Comment by khenderick** [comment]
@openmotics-ci test gateway=bugfix/metrics-interval-fixes frontend=develop

**Comment by khenderick** [comment]
@openmotics-ci test gateway=bugfix/metrics-interval-fixes frontend=develop

**Comment by khenderick** [comment]
Finally approved :relieved:. It was a bad timing to make this PR while we were migrating the CI environment :smile:.

File: testing/unittests/gateway_tests/metrics_tests.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        MetricsTest.intervals = {}
+        config_controller, metrics_controller = self._get_controller(intervals=['energy'])
+        self.assertEqual(MetricsTest.intervals.get('energy'), 300)
+        metrics_controller.set_cloud_interval('energy', 900)
+        self.assertEqual(MetricsTest.intervals.get('energy'), 900)
+        self.assertEqual(config_controller.get_setting('cloud_metrics_interval|energy'), 900)
+
+
+if __name__ == "__main__":
+    unittest.main(testRunner=xmlrunner.XMLTestRunner(output='../gw-unit-reports'))
```
Comment (svanscho) [inline testing/unittests/gateway_tests/metrics_tests.py]: Would be nice to have a more managed solution that scans the full directory for all unit tests (e.g. all of type unittest.TestCase)

**Pull request merged**